includes/class-debug.php

```diff
@@ -58,29 +66,101 @@ public static function log( $message, $type = 'string' ) {
 				error_log( 'AspireUpdate - Could not open or write to the file system. Check file system permissions to debug log directory.' );
 				// phpcs:enable
 			}
-			return;
+			return false;
 		}
+		return true;
+	}
 
-		$timestamp         = gmdate( 'Y-m-d H:i:s' );
-		$formatted_message = sprintf(
-			"[%s] [%s]: %s\n" . PHP_EOL,
-			$timestamp,
-			strtoupper( $type ),
-			self::format_message( $message )
-		);
-
-		$file_path = WP_CONTENT_DIR . '/' . self::$log_file;
+	/**
+	 * Get the content of the log file truncated upto N number of lines.
+	 *
+	 * @param integer  $limit Max no of lines to return. Defaults to a 1000 lines.
```

Comment:
```suggestion
	 * @param integer $limit Max no of lines to return. Defaults to a 1000 lines.
```
Minor: Extra space was there.